staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go

```diff
@@ -143,7 +148,7 @@ func NewCmdCreateSecretGeneric(f cmdutil.Factory, ioStreams genericiooptions.IOS
 	cmd := &cobra.Command{
 		Use:                   "generic NAME [--type=string] [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run=server|client|none]",
 		DisableFlagsInUseLine: true,
-		Short:                 i18n.T("Create a secret from a local file, directory, or literal value"),
+		Short:                 i18n.T("Create a secret from a local file, directory, or literal value. Using generic subcommand indicate an Opaque secret. When used with empty --type flag, the command creates an empty secret of type Opaque."),
```

Comment:
yes , i think this is more neat . I'll change the wordings

Reply:
for clarification not only when not specified but also when we use `generic` in the create command, the secret type will be opaque

Reply:
Right yeah because this is for the generic subcommand, so that should be fine. I think we don't need to specify that the generic subcommand is required for this in the help text.

Reply:
i tried without generic and the secret didn't get created. there was an error

Reply:
we need to mention `generic` as when I tried without it it showed 
``` shell
$ kubectl create secret my-empty
```

```console
error: unknown command "my-empty"
See 'kubectl create secret -h' for help and examples
```

Reply:
So this help text is nested within the generic subcommand and therefore it is not necessary to mention the command since it is already being passed into the cli by the user, so they are aware that this if for the generic subcommand.

Reply:
Sorry I think I'm being unclear, I think the help text here should read `The default type for a secret is Opaque unless otherwise specified by the --type flag.` because there's no need to specify that the help text pertains to the `generic` subcommand, since the user has already typed the subcommand, so they should know that the help text applies to the subcommand and there's no need to restate that.